pkg/apis/kubedirector/v1beta1/decode.go

```diff
@@ -36,7 +36,7 @@ func (setupPackage *SetupPackage) UnmarshalJSON(
 		return nil
 	}
 
-	if err := json.Unmarshal(data, &setupPackage.PackageURL); err != nil {
+	if err := json.Unmarshal(data, &setupPackage.Info); err != nil {
```

Comment:
There are a bunch of mechanical changes in the code to reflect that the configpackage object now contains TWO things (URL string and boolean flag) rather than just the URL. Pretty much any change you see that is substituting "Info" for "PackageURL" is because of that.

Reply:
👍